foundationdb/src/tuple/mod.rs

```diff
@@ -65,28 +85,40 @@ impl DerefMut for Tuple {
 /// For types that are encodable as defined by the tuple definitions on FoundationDB
 pub trait Encode {
     /// Encodes this tuple/elemnt into the associated Write
-    fn encode<W: Write>(&self, _w: &mut W) -> std::io::Result<()>;
+    fn encode<W: Write>(&self, _w: &mut W, tuple_depth: TupleDepth) -> std::io::Result<()>;
+
+    /// Encodes this tuple/elemnt into the associated Write
+    fn encode_to<W: Write>(&self, w: &mut W) -> std::io::Result<()> {
+        self.encode(w, TupleDepth::new())
+    }
+
     /// Encodes this tuple/elemnt into a new Vec
```

Comment:
While you are at it, would you be so kind to fix these typos? Notice `elemnt` in this comment and the other 2 before it.

Reply:
Yep.